index.html

```diff
@@ -24,10 +24,19 @@
                         <h1 class="logo-text">My Logo</h1>
                     </a>
                 </div>
-                <div class="menu-icon">
-                    <a href="">
-                        <img src="/Assets/Icon - Menu.png" alt="Click Menu">
-                    </a>
+
+                <div>
+                    <ul class="nav-pages nav-menu">
+                        <li><a href="#">Portfolio</a></li>
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                    
+                    <div class="menu-icon">
+                        <a href="">
+                            <img src="/Assets/Icon - Menu.png" alt="Click Menu">
+                        </a>
+                    </div>
```

Comment:
- [ ] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)